### PR TITLE
keepass-keepassotpkeyprov: init at 2.6

### PR DESCRIPTION
**Pull request by Enteee**
###### Motivation for this change

From the [Keepass plugin doc](https://keepass.info/plugins.html#otpkeyprov):
> OtpKeyProv is a key provider based on one-time passwords. After protecting your database using this plugin, you need to generate and enter one-time passwords in order to open your database.
>
> All generator tokens that follow the OATH HOTP standard (RFC 4226) are supported.

Template for this change was #23893

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -1447,6 +1447,11 @@
     github = "endgame";
     name = "Jack Kelly";
   };
+  ente = {
```
Comment (JohnAZoidberg) [inline maintainers/maintainer-list.nix]: I'd prefer if it was the same as your GitHub username. Then we wouldn't have to do the mapping, to look up your account.

Comment (Enteee) [reply maintainers/maintainer-list.nix]: I did change that.

File: pkgs/applications/misc/keepass-plugins/keepassotpkeyprov/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{ stdenv, buildEnv, fetchzip, mono }:
+
+let
+  version = "2.6";
+  drv = stdenv.mkDerivation {
+    name = "keepassotpkeyprov-${version}";
```
Comment (c0bw3b) [inline pkgs/applications/misc/keepass-plugins/keepassotpkeyprov/default.nix]:
```suggestion
    pname = "otpkeyprov";
    inherit version;
```

The plugin is listed as just `otpkeyprov` on KeePass website.
And the resulting attribute name [as listed on NixOS.org](https://nixos.org/nixos/packages.html?channel=nixpkgs-unstable&query=keepass-k) will then be `keepass-otpkeyprov`

**Review by c0bw3b — COMMENTED** [review]
Left a little comment.
@Enteee could you also do your addition to maintainers list in a separate commit `maintainers: add Enteee` please ?

Then I'll see this finally gets merged. :)

File: pkgs/applications/misc/keepass-plugins/keepassotpkeyprov/default.nix (hunk 2 above)
Comment (Enteee) [reply pkgs/applications/misc/keepass-plugins/keepassotpkeyprov/default.nix]: done

**Review by c0bw3b — APPROVED** [review]

**Comment by c0bw3b** [comment]
Here we go! :rocket: 
Thanks @Enteee for your patience & availability to see this through.

**Comment by Enteee** [comment]
Thank you @JohnAZoidberg , @c0bw3b  for reviewing & merging.

**Pull request merged**